tests/conftest.py

```diff
@@ -276,10 +276,10 @@ def getReportingToken(self, market, payoutDistribution):
         reportingToken = ABIContract(self.chain, ContractTranslator(ContractsFixture.signatures['ReportingToken']), reportingTokenAddress)
         return reportingToken
 
-    def getChildBranch(self, parentBranch, market, payoutDistribution):
+    def publicGetChildBranch(self, parentBranch, market, payoutDistribution):
```

Comment:
I'm guessing this was a find/replace misstep?

Reply:
I went ahead and changed this to match the function it calls actually.